<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>806</width>
    <height>502</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Mask Detector for 해커톤</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="lbl_img">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>640</width>
      <height>480</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>380</y>
      <width>131</width>
      <height>101</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt; font-weight:600; color:#00aaff;&quot;&gt;Mask Detector&lt;/span&gt;&lt;/p&gt;&lt;p&gt;팀 : 냐옹이&lt;/p&gt;&lt;p&gt;개발 : 김영수 김희규&lt;/p&gt;&lt;p&gt;멘토 : 임도형&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>10</y>
      <width>141</width>
      <height>361</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QPushButton" name="btn_camera">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>10</y>
       <width>100</width>
       <height>90</height>
      </rect>
     </property>
     <property name="toolTip">
      <string extracomment="실시간 카메라 탐지"/>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>resource/UI/camerabutton.png</normaloff>resource/UI/camerabutton.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>96</width>
       <height>96</height>
      </size>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_av">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>130</y>
       <width>100</width>
       <height>90</height>
      </rect>
     </property>
     <property name="toolTip">
      <string extracomment="동영상 탐지"/>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>resource/UI/videobutton.png</normaloff>resource/UI/videobutton.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>96</width>
       <height>96</height>
      </size>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_close">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>250</y>
       <width>100</width>
       <height>90</height>
      </rect>
     </property>
     <property name="toolTip">
      <string extracomment="영상 닫기"/>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>resource/UI/closebutton.png</normaloff>resource/UI/closebutton.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>96</width>
       <height>96</height>
      </size>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>100</y>
       <width>81</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>실시간 카메라</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>220</y>
       <width>81</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>동영상 실행</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>340</y>
       <width>81</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>영상 닫기</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </widget>
  </widget>
  <action name="action11111">
   <property name="text">
    <string>11111</string>
   </property>
  </action>
  <action name="action222222">
   <property name="text">
    <string>222222</string>
   </property>
  </action>
  <action name="action33333">
   <property name="text">
    <string>33333</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
